util/misc.go

```diff
@@ -107,7 +109,7 @@ func SyntaxError(err error) error {
 	if err == nil {
 		return nil
 	}
-	log.Errorf("%+v", err)
+	log.Error(err.Error())
```

Comment:
Do we add some information for this log?

Reply:
I think no need for it. We could improve the message in parser repo.

Reply:
maybe better use `zap.Error`, our log analysis tool maybe need "error" key to identity error message part